README.md

```diff
@@ -7,11 +7,11 @@ The journey begins.
 
 ## Description
 
-Kepler is a stargazing tool—not for those lovely celestial orbs of light, but for your stars you make on GitHub and GitLab.
+Kepler is a stargazing tool—not for those lovely celestial orbs of light, but for your stars on GitHub, GitLab, and beyond.
 
-GitHub stars are great—they let you bookmark interesting projects to view later. What is *not* great is GitHub's interface for managing your stars. If you're anything like [alex0112](https://github.com/alex0112), sorting through your 1k+ stars is, well, impossible on GitHub. This project aims to be a better way to manage your stars so you can keep up on all your favorite projects.
+GitHub stars are great-- they let you bookmark interesting projects to view later. What is *not* great is GitHub's interface for managing your stars. If you're anything like [alex0112](https://github.com/alex0112), sorting through your 1k+ stars is, well, impossible on GitHub. This project aims to be a better way to manage your stars so you can keep up on all your favorite projects.
 
-*More details to come…*
```

Comment:
Typographic nit picks: revert `--` back to an m-dash `—`, and change the `...` to a proper ellipsis `…`.